### PR TITLE
Re-host dependencies and update URLs to highfidelity.com

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by Atlante45** [comment]
## Test Plan:
- No test needed, if it builds, it's good

[ticket](https://highfidelity.manuscript.com/f/cases/16635/LibOVR-external-does-not-download-on-macOS)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11298/HighFidelity-Beta-PR13606-2843292.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11298/HighFidelity-Beta-PR13606-2843292.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11298/HighFidelity-Beta-Interface-PR13606-2843292.dmg

File: cmake/externals/openvr/CMakeLists.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ string(TOUPPER ${EXTERNAL_NAME} EXTERNAL_NAME_UPPER)
 
 ExternalProject_Add(
   ${EXTERNAL_NAME}
-  URL https://github.com/ValveSoftware/openvr/archive/v1.0.6.zip
+  URL https://public.highfidelity.com/dependencies/v1.0.6.zip
```
Comment (wayne-chen) [inline cmake/externals/openvr/CMakeLists.txt]: Looks like this is a bad link.

Comment (wayne-chen) [reply cmake/externals/openvr/CMakeLists.txt]: Should be `https://public.highfidelity.com/dependencies/openvr-1.0.6.zip`.

Comment (Atlante45) [reply cmake/externals/openvr/CMakeLists.txt]: Thanks @wayne-chen!

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11313/HighFidelity-Beta-PR13606-5a80eb6.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11315/HighFidelity-Beta-PR13606-f038ba9.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11315/HighFidelity-Beta-PR13606-f038ba9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11315/HighFidelity-Beta-Interface-PR13606-f038ba9.exe

**Pull request merged**